tests/chains/tendermint/simapp/export.go

```diff
@@ -71,12 +74,19 @@ func (app *SimApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []
 
 	// withdraw all validator commission
 	app.StakingKeeper.IterateValidators(ctx, func(_ int64, val stakingtypes.ValidatorI) (stop bool) {
-		_, _ = app.DistrKeeper.WithdrawValidatorCommission(ctx, val.GetOperator())
+		valBz, err := app.StakingKeeper.ValidatorAddressCodec().StringToBytes(val.GetOperator())
```

Comment:
At L77 you used `app.StakingKeeper.ValidatorAddressCodec().StringToBytes` to convert `val.GetOperator()` into `sdk.ValAddress`.
On the other hand, at L115, you used `sdk.ValAddressFromBech32` for the same purpose.

What is difference between them? Which one is the best choice? Why is it so?

Reply:
According to discussion, the simapp from ibc-go@v8 also uses different methods for conversion.
Therefore you need not fix this code. Thank you.